src/components/forms/send-two-fa-code-form.tsx

```diff
@@ -110,4 +126,22 @@ const ContactChoiceForm = styled(Form)`
   }
 `;
 
+const InputRadioWrapper = styled(Label)`
```

Comment:
If you have better naming idea, don't hesitate to let me know 😄

Reply:
Nope 😆 .